dbt_subprojects/dex/macros/models/_decoding/uniswap_exposed_blockchains_list.sql

```diff
@@ -0,0 +1,48 @@
+{% macro uniswap_old_blockchains_list() %}
+   {{ return([
+       "arbitrum"
+       ,"avalanche_c"
+       , "base"
+       , "blast"
+       , "bnb"
+       , "bob"
+       , "boba"
+       , "celo"
+       , "degen"
+       , "ethereum"
+       , "fantom"
+       , "flare"
+       , "gnosis"
+       , "kaia"
+       , "linea"
+       , "mantle"
+       , "mode"
+       , "nova"
+       , "polygon"
+       , "ronin"
+       , "scroll"
+       , "sei"
+       , "sonic"
+       , "sophon"
+       , "unichain"
+       , "viction"
+       , "worldchain"
+       , "zkevm"
+       , "zksync"
+       , "zora"
+       
+       
+    ]) }}
+{% endmacro %} 
+
+
+{% macro uniswap_new_blockchains_list() %}
+   {{ return([
+      "berachain"
+       , "sonic"
+       , "unichain"
+    ]) }}
+{% endmacro %} 
+
+
+-- "optimism" has some problems with traces, need to fix later
```

Comment:
def seems like a blocker for full implementation

Reply:
ye it's a solvable problem though, it's just the usual problem of pool initialization that exists for op chains